models/modelling/commissioning/activities/int_comm_ambulatory_sensitive_nel.sql

```
@@ -0,0 +1,29 @@
+    select *, left(primary_diagnosis_code, 3) as clipped_primary_diagnosis_code
+    from {{ ref('int_sus_apc_encounter') }}
+    where left(spell_admission_method, 1) = '2' -- Non-elective - emergency
+)
+
+
+SELECT be.visit_occurrence_id
+    , be.sk_patient_id
+    , be.source
+    , be.organisation_id
+    , be.start_date
+    , 1 as ambulatory_sensitive_nel, sec.term as primary_diagnosis_term
```

SQLFluff (4.2.2)

[error] Select targets should be on a new line unless there is only one select target.

(LT09)